.github/workflows/build-test.yml

```diff
@@ -0,0 +1,52 @@
+name: go-ethereum workflow
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '**'
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
is there a specific reason this doesn't run on `self-hosted` as well?

Reply:
No, my mistake. I stopped playing around with it at some point. Will pick it back up today